src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin: 2px 6px 2px 2px;
```

Comment:
Try margin-right of 4px instead of 6px. I think this `margin-right: 4px; `should work.